src/Redux/api.tsx

```diff
@@ -261,7 +266,7 @@ const routes = {
   },
 
   getOnlineDoctors: {
-    path: "/api/v1/users/?user_type=Doctor&ordering=-last_login",
+    path: "/api/v1/users/?user_type=Doctor&ordering=-last_login/",
```

Comment:
1. Why is this having a trailing slash? The API endpoint here is `/api/v1/users/` And `?user_type=...&...` are the search params passed to that endpoint. **You shouldn't be adding a trailing slash after search params. Add only for endpoints.**

2. **Also, undo adding trailing slashes for API's that are not under the scope of the migration**. The reason is, that these API routes are also used by fireRequest as of now (for the parts that are not migrated yet), and `path` may not have trailing slash when being used by fireRequest since an arg called `path: string[]` passed to fireRequest is appended at the end of the path specified here and fireRequest expects the path not to have a trailing slash since it'll add it.